test/httpoison_test.exs

```diff
@@ -94,6 +94,17 @@ defmodule HTTPoisonTest do
     assert_response HTTPoison.get("https://localhost:8433/get", [], ssl: [cacertfile: cacert_file, keyfile: key_file, certfile: cert_file])
   end
 
+  test "http+unix scheme" do
+    if Application.get_env(:httparrot, :unix_socket, false) do
```

Comment:
Just like your ssl support I've set the default to `true` in httparrot. It will be false for versions of httparrot that do not support unix sockets yet.